grails-app/views/order/list.gsp

```diff
@@ -153,7 +153,8 @@
 							</tbody>
 							<tfoot>
 								<tr class="odd">
-									<th colspan="12"></th>
+									<g:set var="colspan" value="${params.orderTypeCode == OrderTypeCode.PURCHASE_ORDER ? 12 : 8}"/>
```

Comment:
It's things like this that make me want to rethink how we do the UI for order. Either separate into separate directories

transferOrder/list
purchaseOrder/list

Or make it a wholesale change by adding PurchaseOrder extends Order, etc.

Reply:
Not for this ticket but should discuss and ticket our approach in a tech huddle.

Reply:
@jmiranda I think it would be better to have a more generic Order domain and a separate PurchaseOrder extending it.

Reply:
Yes, I agree. That is something that we should discuss during tech huddle.

Reply:
Cool, I've added that to the tech huddle notes so we can bring it up on Monday if we have time or add another tech huddle to discuss.